pkg/rpc/loadbalancer/client.go

```diff
@@ -0,0 +1,127 @@
+package loadbalancer
+
+import (
+	"time"
+
+	"github.com/docker/infrakit/pkg/plugin"
+	rpc_client "github.com/docker/infrakit/pkg/rpc/client"
+	"github.com/docker/infrakit/pkg/spi/loadbalancer"
+)
+
+// clientResult implements loadbalancer.Result
+type clientResult string
+
+func (f clientResult) String() string {
+	return string(f)
+}
+
+// NewClient returns a plugin interface implementation connected to a plugin
+func NewClient(name plugin.Name, socketPath string) (loadbalancer.L4, error) {
```

Comment:
I think we can leave them as they are without the L4 qualification.  When we come up with L7 we can reuse the types when we can. If we need to qualify the types we will just create packages like pkg/spi/loadbalancer/l4 and pkg/spi/loadbalancer/l7.   So we don't need to add L4 in the names yet.